x/feegrant/client/cli/tx_test.go

```diff
@@ -613,7 +613,7 @@ func (s *CLITestSuite) msgSubmitLegacyProposal(clientCtx client.Context, from, t
 	}
 
 	args := append([]string{
-		fmt.Sprintf("--%s=%s", govcli.FlagTitle, title),               //nolint:staticcheck // SA1019: govcli.FlagTitle is deprecated: use FlagTitle instead
+		fmt.Sprintf("--%s=%s", govcli.FlagTitle, title),
 		fmt.Sprintf("--%s=%s", govcli.FlagDescription, description),   //nolint:staticcheck // SA1019: govcli.FlagDescription is deprecated: use FlagDescription instead
```

Comment:
slightly unrelated but these comments are weird, there is nothing to use instead.